Mage/src/main/java/mage/abilities/effects/common/ConjureCardEffect.java

```diff
@@ -74,6 +74,9 @@ public boolean apply(Game game, Ability source) {
 
     @Override
     public String getText(Mode mode) {
+        if (staticText != null && !staticText.isEmpty()) {
```

Comment:
@xenohedron empty text uses to hide inner effects/abilities in some use cases. Example:

![shot_230827_064835](https://github.com/magefree/mage/assets/8344157/c8d5207c-3e54-422b-9fde-a0168efb4115)

Reply:
I compared the full check before and after and didn't find any issues.

Reply:
This change aligns these effect classes with the many (100+) existing that already check 

```
    @Override
    public String getText(Mode mode) {
        if (staticText != null && !staticText.isEmpty()) {
            return staticText;
        }
```

In general, setting empty staticText won't help. You need to adjust the text in some other way, typically by splitting the desired text across the effects.